docs/modules/ROOT/partials/getting-started/proc-setting-up-kafka-streams-storage.adoc

```diff
@@ -75,16 +75,38 @@ spec:
     streams:
       bootstrapServers: "my-cluster-kafka-bootstrap.my-project.svc:9092"
 ----
-      
+
 . Click *Create* and wait for the {registry} route to be created on OpenShift.
 
 . Click *Networking* > *Route* to access the new route for the {registry} web console. For example:
 +
 [source]
 ----
-http://example-apicurioregistry.my-project.my-domain-name.com/   
+http://example-apicurioregistry.my-project.my-domain-name.com/
 ----
 
+== Overriding default Kafka topic names
```

Comment:
Using the modular docs approach, procedure topics are meant to be atomic (i.e., only contain ".Prerequisites" and ".Procedure" headings). Suggest breaking this subheading out into its own procedure topic.